grow_cluster_test.py

```diff
@@ -60,10 +60,10 @@ def get_stress_cmd(self, mode='write', duration=None):
         :rtype: basestring
         """
         ip = self.db_cluster.get_node_private_ips()[0]
-        population_size = self.params.get('cassandra_stress_population_size', default=1000000)
+        population_size = self.params.get('cassandra_stress_population_size') or 1000000
         if not duration:
-            duration = self.params.get('test_duration', default=60)
-        threads = self.params.get('cassandra_stress_threads', default=1000)
+            duration = self.params.get('test_duration')
+        threads = self.params.get('cassandra_stress_threads') or 1000
```

Comment:
why do you keep defaults? it is set in `test-cases/features/scale-cluster.yaml`

Reply:
same as above